frontend/src/Context/PreviewPublisherProvider/usePreviewPublisher/usePreviewPublisher.tsx

```diff
@@ -172,7 +186,10 @@ const usePreviewPublisher = (): PreviewPublisherContextType => {
   }, []);
 
   const addPublisherListeners = useCallback(
-    (publisher: Publisher) => {
+    (publisher: Publisher | null) => {
+      if (!publisher) {
+        return;
+      }
```

Comment:
I'm not sure why TS didn't catch this, but this can be `null` when we see a publishing error.

https://github.com/Vonage/vonage-video-react-app/blob/a0c2287fd2fcf55b5143f99b9180362b68d6c9a1/frontend/src/Context/PreviewPublisherProvider/usePreviewPublisher/usePreviewPublisher.tsx#L195-L203